src/UI/Root/PrimaryPageManager.tsx

```diff
@@ -139,7 +152,21 @@ export class PrimaryPageManager implements PageManager {
     };
   }
 
+  private filterRestricedPages(): Page[] {
+    return Object.values(
```

Comment:
this chaining of methods is losing me a bit to be honest. I would have seen another approach instead, I'd only add the pages to `this.pageDisctionary`, if the configFile with the features contains it. The reason why, is because we might have more of these pages in the future, and adding filter requirements in here is complicating things even more, and I'm not even talking about the chaining of Object methods to even be able to filter them out.

Reply:
Good point, I wanted to avoid having pageDictionary values to be Page | null and also wanted to avoid reworking route types, but it's better now than later :D